easybuild/easyconfigs/r/Ruby-bundle/Ruby-bundle-2.3.4-Ruby-2.3.4.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'Ruby-bundle'
+version = '2.3.4'
+versionsuffix = '-Ruby-%s' % version
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# sources = [SOURCELOWER_TAR_GZ]
+# source_urls = ['http://cache.ruby-lang.org/pub/ruby/']
+
+dependencies = [('Ruby', version)]
+
+exts_defaultclass = 'RubyGem'
+ext_options = {
+    'source_urls': ['http://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
+modextrapaths = {'GEM_HOME': ''}
+
+exts_list = [
+    ('archive-tar-minitar', '0.5.2', ext_options),
```

Comment:
not sure how to add checksum with ext_list. I didn't see any examples with R-bundle